js/image/Index.svelte

```diff
@@ -122,7 +122,7 @@
 
 		<ImageUploader
 			bind:active_tool
-			bind:value={_value}
+			bind:value
```

Comment:
What was happening is that we were binding to `_value` so the `value` prop (which is what gets passed to events) was never updated during the upload.

I think this is the right thing to do. By binding to value, _value will automatically be updated. Also the ImageUploaded calls `normalise_file` so value will have the right url

Reply:
Both of the test cases from #6322 work with this change as far as I can tell

<img width="1322" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/a13927dc-07f3-4ac8-ae34-c83ad15a06ca">

<img width="1373" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/a3efae3f-2ae4-4f1c-87e2-86dd53d9e437">